### ISSUE TITLE
Book app docker images should move out of personal acct

### ISSUE THREAD

**Issue by larryms** [Microsoft/MTC_ContainerCamp#75]
The book app is currently referencing images in the ravitella account:
ravitella/docker-springboot-readinglistapplication:latest
ravitella/docker-springboot-recommendationservice:latest

This should be changed to either a generic (MTC or Microsoft) github account or to use a OneMTC ACR image repository.

**Comment by amazaheri** [comment]
@RaviTella can you push this to MTC Repos or perhaps push the image to our MTC Org Docker Repository?

**Comment by amazaheri** [comment]
@RaviTella any update on this or should we push this to next release cycle?

**Comment by RaviTella** [comment]
I have pushed the images to the following repos: 
onemtc/springboot-recommendationservice
onemtc/springboot-readinglistapplication

I have created a new Docker hub account with id "onemtc". I can also push it to any other Docker Hub account. Let me know the credentials for the account you would like me to use. 

Remember all the deploymnet files need to be updated with the new image repos. I personally feel this is not worth it :)
